<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KOnlineBankingSetupWizard</class>
 <widget class="QWizard" name="KOnlineBankingSetupWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Online Banking Account Setup</string>
  </property>
  <widget class="QWizardPage" name="FIPage">
   <property name="title">
    <string>Select Financial Institution</string>
   </property>
   <layout class="QHBoxLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Please select your financial institution from the list below...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="m_selectionTab">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="autoTab">
         <attribute name="title">
          <string comment="@title type of online banking connection">Automatic</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="m_listFi"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="manualTab">
         <attribute name="title">
          <string comment="@title type of online banking connection">Manual</string>
         </attribute>
         <layout class="QVBoxLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="textLabel1_5">
              <property name="text">
               <string>FID</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel4">
              <property name="text">
               <string>Org</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="KLineEdit" name="m_userAgent"/>
            </item>
            <item row="0" column="1">
             <widget class="KLineEdit" name="m_bankName"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>UserAgent</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KLineEdit" name="m_fid"/>
            </item>
            <item row="2" column="1">
             <widget class="KUrlRequester" name="m_url"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="textLabel2_4">
              <property name="text">
               <string>URL</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="spacer3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>90</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="LoginPage">
   <property name="title">
    <string>Enter Login Details</string>
   </property>
   <layout class="QGridLayout" name="gridLayout1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="qLabel1">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Please enter the username and password you use to log into this bank for online banking.  Please note that many banks require a separate signup, and assign a separate PIN or password just for online banking from home.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="textLabel1_4">
      <property name="text">
       <string comment="@label account username">Username</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="KLineEdit" name="m_editUsername"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="textLabel2_3">
      <property name="text">
       <string>Password</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="KLineEdit" name="m_editPassword"/>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="m_storePassword">
      <property name="text">
       <string>Store password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="textLabel1_2_3">
      <property name="text">
       <string>Identify as</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <item>
       <widget class="KComboBox" name="m_applicationCombo"/>
      </item>
      <item>
       <widget class="KLineEdit" name="m_applicationEdit">
        <property name="whatsThis">
         <string>Expert option to enter the four digit version information of the application</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer4_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="textLabel1_6">
      <property name="text">
       <string>Header Version</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="KComboBox" name="m_headerVersionCombo"/>
      </item>
      <item>
       <widget class="KLineEdit" name="m_headerVersionEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QTextBrowser" name="m_textDetails">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="m_clientUidLabel">
      <property name="text">
       <string>Client UID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="KLineEdit" name="m_editClientUid"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="textLabel3">
      <property name="text">
       <string>Connection Details</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="KMessageWidget" name="m_problemMessages"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="AccountPage">
   <property name="title">
    <string>Select Account</string>
   </property>
   <layout class="QHBoxLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>Please select the account from your financial institution from the list below which matches this account.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="m_listAccount">
        <column>
         <property name="text">
          <string>Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bank</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Branch</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="WizardPage">
   <property name="title">
    <string>Finish</string>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="m_labelFinal">
        <property name="text">
         <string>Congratulations: you have successfully set up your bank for online banking via OFX.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_selectionTab</tabstop>
  <tabstop>m_listFi</tabstop>
  <tabstop>m_bankName</tabstop>
  <tabstop>m_fid</tabstop>
  <tabstop>m_url</tabstop>
  <tabstop>m_userAgent</tabstop>
  <tabstop>m_editUsername</tabstop>
  <tabstop>m_editPassword</tabstop>
  <tabstop>m_editClientUid</tabstop>
  <tabstop>m_storePassword</tabstop>
  <tabstop>m_applicationCombo</tabstop>
  <tabstop>m_applicationEdit</tabstop>
  <tabstop>m_headerVersionCombo</tabstop>
  <tabstop>m_headerVersionEdit</tabstop>
  <tabstop>m_listAccount</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
